docs/whatsnew/0.8.rst

```diff
@@ -0,0 +1,140 @@
+.. doctest-skip-all
+
+.. _whatsnew-0.8:
+
+**************************
+What's New in SunPy 0.8?
+**************************
```

Comment:
@bsipocz Thanks, I suspect a pass over all rst files are in order down the road. 

But for now @wafels, ignore my comment here. Not an issue for 0.8!